bin/checkshrinkwrap.js

```diff
@@ -51,10 +51,11 @@ function checkProject(pathToProject) {
     const packageDevDeps = packages.devDependencies;
 
     checkImpreciseDependencies(packageDeps);
-    checkImpreciseDependencies(packageDevDeps);
+    //checkImpreciseDependencies(packageDevDeps);
     checkDuplicateDependencies(packageDeps, packageDevDeps);
 
-    const allDeps = Object.assign({}, packageDeps, packageDevDeps);
+    //const allDeps = Object.assign({}, packageDeps, packageDevDeps);
```

Comment:
Hmm, so devDeps are not covered anymore?

Reply:
devDeps don't seem to get installed into shrinkwrap, so checking fails.

Reply:
http://blog.bithound.io/npm-shrinkwrap-your-devdependencies-too/ it does if you do `npm shrinkwrap --dev `

Reply:
ah oops! let me recreate them :)